jekyll/_cci2_ja/faq.md

```diff
@@ -28,76 +28,76 @@ CircleCI の従業員がお客様の許諾を得ずにコードを見ること
 ## 移行
 {: #migration }
 
-Visit the [Migration Introduction]({{ site.baseurl }}/2.0/migration-intro/) doc which links out to migration guides for various platforms.
+[移行の概要]({{ site.baseurl }}/ja/2.0/migration-intro/)ドキュメントをご覧ください。さまざまなプラットフォーム用の移行ガイドへのリンクにアクセスできます。
 
 ### Jenkins から CircleCI  へ移行する方法を教えてください。
 {: #how-do-i-migrate-from-jenkins-to-circleci }
 {:.no_toc}
 
-Jenkins と CircleCI のコンセプトの違いについては「[Jenkins からの移行]({{ site.baseurl }}/2.0/migrating-from-jenkins/)」をご覧ください。
+Jenkins と CircleCI のコンセプトの違いについては「[Jenkins からの移行]({{ site.baseurl }}/ja/2.0/migrating-from-jenkins/)」をご覧ください。
 
-### Can I use CircleCI without creating base images?
+### 基本イメージを作成していなくても、CircleCI  を使用できますか?
 {: #can-i-use-circleci-without-creating-base-images }
 {:.no_toc}
 
-Yes, CircleCI provides a selection of "convenience images" for use with the Docker executor. For a full list, along with usage instructions, visit the [CircleCI Developer Hub](https://circleci.com/developer/images) and the [CircleCI Images guide]({{site.baseurl}}/2.0/circleci-images/).
+はい、CircleCI では Docker Executor と共に使用する多数の「CircleCI イメージ」を提供しています。 使用方法および全リストは、[CircleCI Developer Hub](https://circleci.com/developer/images) および [CircleCI イメージガイド]({{site.baseurl}}/2.0/circleci-images/)をご覧ください。
 
-For the `machine` executor, see the [available machine images](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images) list.
+`machine` Executor に関しては、[利用可能なマシンイメージ](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images)のリストをご覧ください。
```

Comment:
this should probably be updated not to use hardcoded links but we (I) will make the change in a separate PR since this it not about localization.

```suggestion
`machine` Executor に関しては、[利用可能なマシンイメージ]({{ site.baseurl }}/ja/2.0/configuration-reference/#available-machine-images)のリストをご覧ください。
```